validator/src/commands/wait_for_restart_window/mod.rs

```diff
@@ -47,7 +87,6 @@ pub(crate) fn command(default_args: &DefaultArgs) -> App<'_, '_> {
                 .long("max-delinquent-stake")
                 .takes_value(true)
                 .validator(is_valid_percentage)
-                .default_value(&default_args.wait_for_restart_window_max_delinquent_stake)
```

Comment:
Whatever approach we land on for `min_idle_time`, we should do the same for `max_delinquent_stake`